profile/README.md

```diff
@@ -0,0 +1,26 @@
+# Bottlerocket
+
+Bottlerocket is a Linux-based operating system optimized for hosting containers.
+It’s free and open-source software, developed in the open here on GitHub.
+Bottlerocket is installed as the base operating system on the machine or instance where your containers themselves are running.
+It is specifically designed to work with your container orchestrator (like Kubernetes) to automate the lifecycle of the containers running in your cluster.
+Bottlerocket runs in the cloud or in your datacenter.
+
+## Project Navigation
+
+The `bottlerocket-os` GitHub organization contains a number of repos. Feel free to examine everything, but most people are looking for the following:
+
+- [`bottlerocket-os/bottlerocket`](https://github.com/bottlerocket-os/bottlerocket) the source to the operating system and associated tools.
```

Comment:
"the source to" / "the source for" shows up in every description - might be OK to omit

Reply:
Fair enough. I went back and forth on that one as well.